Support for enable_language(hip) or project( ... hip)

It looks like the cmake capabilities in chipStar don't include support for enable_language(hip) or project( ... hip).

For example, give the code:

```
> cat saxpy_hip.hip
#include <stdio.h>
#include "hip/hip_runtime.h"

int main(void)
{
}
> cat CMakeLists.txt 
cmake_minimum_required(VERSION 3.10)

# Search for rocm in common locations
list(APPEND CMAKE_MODULES_PREFIX $ENV{HIP_PATH}/cmake)
list(APPEND CMAKE_PREFIX_PATH $ENV{HIP_PATH})

# Find hip
project(MY_PROJECT LANGUAGES HIP CXX)

# Set compiler and linker
set(CMAKE_BUILD_TYPE Debug)

# Create the excutable. since extension is .hip it should work
add_executable(saxpy_hip saxpy_hip.hip)
```

Then trying to build:
```
> mkdir build
> cd build
> cmake  ..
-- The HIP compiler identification is unknown
CMake Error at /opt/aurora/25.190.0/spack/unified/0.10.1/install/linux-sles15-x86_64/gcc-13.3.0/cmake-3.31.8-mvhtj5u/share/cmake-3.31/Modules/CMakeDetermineHIPCompiler.cmake:174 (message):
  Failed to find ROCm root directory.
Call Stack (most recent call first):
  CMakeLists.txt:8 (project)
```

I then tried to set `CMAKE_HIP_COMPILER_ROCM_ROOT` to the chipStar path, but this then was looking for ` /home/bertoni/latest/20251117-latest-release/lib/cmake/hip-lang/hip-lang-config.cmake`:
```
> cmake  -DCMAKE_CXX_COMPILER=clang++  -DCMAKE_HIP_COMPILER_ROCM_ROOT=$HIP_PATH ..
-- The HIP compiler identification is unknown
CMake Error at /opt/aurora/25.190.0/spack/unified/0.10.1/install/linux-sles15-x86_64/gcc-13.3.0/cmake-3.31.8-mvhtj5u/share/cmake-3.31/Modules/CMakeDetermineHIPCompiler.cmake:217 (message):
  The ROCm root directory:

   /home/bertoni/latest/20251117-latest-release

  does not contain the HIP runtime CMake package, expected at one of:

   /home/bertoni/latest/20251117-latest-release/lib/cmake/hip-lang/hip-lang-config.cmake
   /home/bertoni/latest/20251117-latest-release/lib64/cmake/hip-lang/hip-lang-config.cmake

Call Stack (most recent call first):
  CMakeLists.txt:8 (project)


-- Configuring incomplete, errors occurred!
```

I confirm that the above CMakelists.txt works fine on an AMD system:
```
> cmake ../
-- The HIP compiler identification is Clang 20.0.0
-- The CXX compiler identification is Clang 20.0.0
-- Detecting HIP compiler ABI info
-- Detecting HIP compiler ABI info - done
-- Check for working HIP compiler: /soft/compilers/rocm/rocm-7.0.2/lib/llvm/bin/clang++ - skipped
-- Detecting HIP compile features
-- Detecting HIP compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /soft/compilers/rocm/rocm-7.0.2/lib/llvm/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring done (2.2s)
-- Generating done (0.0s)
-- Build files have been written to: /home/bertoni/hip-training/simple/build
> make VERBOSE=1
/usr/bin/cmake -S/home/bertoni/hip-training/simple -B/home/bertoni/hip-training/simple/build --check-build-system CMakeFiles/Makefile.cmake 0
[...]
[ 50%] Building HIP object CMakeFiles/saxpy_hip.dir/saxpy_hip.hip.o
/soft/compilers/rocm/rocm-7.0.2/lib/llvm/bin/clang++  -D__HIP_ROCclr__=1  -g -O -std=gnu++17 --offload-arch=gfx90a --offload-arch=gfx90a --offload-arch=gfx90a --offload-arch=gfx90a --offload-arch=gfx90a --offload-arch=gfx9\
0a --offload-arch=gfx90a --offload-arch=gfx90a -o CMakeFiles/saxpy_hip.dir/saxpy_hip.hip.o -x hip -c /home/bertoni/hip-training/simple/saxpy_hip.hip
[100%] Linking HIP executable saxpy_hip
/usr/bin/cmake -E cmake_link_script CMakeFiles/saxpy_hip.dir/link.txt --verbose=1
/soft/compilers/rocm/rocm-7.0.2/lib/llvm/bin/clang++ -g -O --offload-arch=gfx90a --offload-arch=gfx90a --offload-arch=gfx90a --offload-arch=gfx90a --offload-arch=gfx90a --offload-arch=gfx90a --offload-arch=gfx90a --offload-arch=gfx90a --hip-link --rtlib=compiler-rt -unwindlib=libgcc CMakeFiles/saxpy_hip.dir/saxpy_hip.hip.o -o saxpy_hip  /soft/compilers/rocm/rocm-7.0.2/lib/libamdhip64.so.7.0.70002
make[2]: Leaving directory '/home/bertoni/hip-training/simple/build'
[100%] Built target saxpy_hip
make[1]: Leaving directory '/home/bertoni/hip-training/simple/build'
/usr/bin/cmake -E cmake_progress_start /home/bertoni/hip-training/simple/build/CMakeFiles 0
```

If there is a way around this, let me know! Thanks!

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -819,6 +819,10 @@ install(
 )
 install(EXPORT hip-targets FILE hip-targets.cmake NAMESPACE hip:: DESTINATION lib/cmake/hip COMPONENT binary)
 
+# Install hip-lang-config.cmake for CMake's native HIP language support
+install(FILES ${CMAKE_SOURCE_DIR}/cmake/hip-lang/hip-lang-config.cmake
+  DESTINATION ${CONFIG_LANG_PACKAGE_INSTALL_DIR})
+
 # Generate hip_version.h
 set(_pchStatus 0)
 set(_versionInfoHeader

